pytripgui/canvas_vc/plotter/managers/placement_manager.py

```diff
@@ -0,0 +1,92 @@
+from matplotlib.gridspec import GridSpec, SubplotSpec
+
+
+class PlacementManager:

```

Comment:
Add a comment describing responsibility of this class

Reply:
Done

Reply:
Done